src/routes/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import PageNotFoundView from '../views/errors/PageNotFoundView';
+import App from '../views/pages/DummyView';
+import HomeView from '../views/pages/HomeView';
+import LoginView from '../views/pages/LoginView';
+
+const AllRoutes = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<HomeView />} />
+      <Route path="/login" element={<LoginView />} />
+      <Route path="/dummy" element={<App />} />
+      <Route path="*" element={<PageNotFoundView />} />
+    </Routes>
+  </Router>
```

Comment:
All the routes needs to be encapsulated in `<BrowserRouter> `

Reply:
Router is an alias of BrowserRouter
Check well on line 2 where I imported them

Reply:
@rukundo-kevin